src/Renci.SshNet/PrivateKeyFile.cs

```diff
@@ -63,7 +63,7 @@ namespace Renci.SshNet
     /// </list>
     /// </para>
     /// </remarks>
-    public class PrivateKeyFile : IDisposable
+    public class PrivateKeyFile : IPrivateKeyFile, IDisposable
```

Comment:
Do we want to make this one sealed to allow the JIT to devirtualize calls?
Not a big deal.